dataprofiler/tests/data_readers/test_data_utils.py

```diff
@@ -37,7 +37,18 @@ def test_file_UTF_encoding_detection(self):
             detected_encoding = \
                 data_utils.detect_file_encoding(file_path=input_file["path"])
             self.assertEqual(detected_encoding.lower(), input_file["encoding"])
-
+        
+        try:
+            from charset_normalizer import CharsetNormalizerMatches as CnM
+            detected_encoding = \
+                data_utils.detect_file_encoding(file_path=os.path.join(test_dir, 'txt/utf8.txt'))
+            self.assertEqual("utf-8", detected_encoding.lower()) 
+        except:
+            # Verify that detected encoding is incorrect using chardet instead of charset_normalizer
+            detected_encoding = \
+                data_utils.detect_file_encoding(file_path=os.path.join(test_dir, 'txt/utf8.txt'))
+            self.assertEqual("windows-1254", detected_encoding.lower()) 
```

Comment:
why don't we use assertIn for a list of utf-8 and windows-1254. So, we don't have to run detect_file_encoding in both try and except.

Reply:
Well, we wouldn't want windows-1254 as a result for charset_normalizer. Instead, it might be a better idea to only do the self.assertEqual in the try and except